app/javascript/src/components/Dashboard/Notes/NewNotePane.jsx

```diff
@@ -1,16 +1,121 @@
 import React from "react";
 
-import { Pane } from "neetoui";
+import { Check } from "@bigbinary/neeto-icons";
+import { Formik, Form } from "formik";
+import { Pane, Typography, Button, Toastr } from "neetoui/v2";
+import { Input, Select } from "neetoui/v2/formik";
+import * as yup from "yup";
 
-import NewNoteForm from "./NewNoteForm";
+import { ROLES } from "./constants";
 
-export default function NewNotePane({ fetchNotes, showPane, setShowPane }) {
+export default function NewNotePane({ setNotes, showPane, setShowPane }) {
   const onClose = () => setShowPane(false);
+  const handleSubmit = values => {
+    setNotes(notes => {
+      const newNote = {
+        id: notes.length + 1,
+        title: values.title,
+        description: values.description,
+        tags: values.tags.map(({ label }) => label),
+        contact: values.tags.map(({ label }) => label)
+      };
+      return [...notes, newNote];
+    });
+    Toastr.success("Note added successfully");
+    onClose();
+  };
   return (
-    <Pane title="Add New Note" isOpen={showPane} onClose={onClose}>
-      <div className="px-6">
-        <NewNoteForm onClose={onClose} refetch={fetchNotes} />
-      </div>
+    <Pane isOpen={showPane} onClose={onClose}>
+      <Pane.Header>
+        <Typography style="h2" weight="semibold">
+          Add New Note
+        </Typography>
+      </Pane.Header>
+      <Formik
+        initialValues={{
+          title: "",
+          description: ""
+        }}
+        onSubmit={handleSubmit}
+        validationSchema={yup.object({
+          title: yup.string().required("Title is required"),
+          description: yup.string().required("Description is required")
+        })}
```

Comment:
Please move `initialValues` and `validationSchema` to a `constants.js` file in the scope of this directory.